packages/web/components/MobileFooter.tsx

```diff
@@ -0,0 +1,175 @@
+import MetaForum from 'assets/drawer/forum.png';
+import MetaPlayers from 'assets/drawer/players.png';
+import MetaQuests from 'assets/drawer/quests.png';
+import MetaRaids from 'assets/drawer/raids.png';
+import MetaGameLogo from 'assets/logo.png';
+import { MetaLink } from 'components/Link';
+import React from 'react';
+
+import { DrawerItems } from './DrawerItems';
+
+const MenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
+  children,
+  href,
+  isExternal,
+}) => {
+  return (
+    <MetaLink zIndex="2" href={href} isExternal={isExternal}>
+      <Button
+        display="flex"
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        width="20vw"
+        height="5rem"
+        textDecoration="none"
+        variant="link"
+        p="1"
+        fontFamily="mono"
+        color="whiteAlpha.700"
+      >
+        {children}
+      </Button>
+    </MetaLink>
+  );
+};
+
+const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
+  children,
+  href,
+  isExternal,
+}) => {
+  return (
+    <MetaLink
+      zIndex="2"
+      href={href}
+      isExternal={isExternal}
+      margin="0 !important"
+      width="7rem"
+      height="7rem"
+    >
+      <Button
+        display="flex"
+        alignItems="center"
+        justifyContent="center"
+        width="6rem"
+        height="6rem"
+        textDecoration="none"
+        variant="link"
+        margin="0.5rem"
+        padding="0.5rem"
+        backgroundImage={`url(${MetaBoxButton})`}
+      >
+        {children}
+      </Button>
+    </MetaLink>
+  );
+};
+
+export interface SubImageProps {
+  src: string;
+  alt: string;
+}
+
+export const SubImage: React.FC<SubImageProps> = ({ src, alt }) => {
+  return <Image src={src} alt={alt} height="85%" />;
+};
+
+export const MobileFooter: React.FC = () => {
+  const [show, setShow] = React.useState(true);
+  const handleToggle = () => setShow(!show);
+
+  return (
+    <Flex
+      as="nav"
+      align="center"
+      justify="center"
+      wrap="wrap"
+      color="offwhite"
+      position="fixed"
+      display={{ base: 'flex', md: 'none' }}
+      left="0"
+      bottom="0"
+      width="100%"
+      height="5rem"
+      zIndex="11"
+      background="linear-gradient(180deg, #40347C 58.55%, #A751BD 100%)"
+    >
+      <MenuItem
+        href="https://forum.metagame.wtf/c/quest/5/l/latest?board=default"
+        isExternal
+      >
+        <Image src={MetaQuests} alt="MetaQuests" />
+        Quests
+      </MenuItem>
+
+      <MenuItem
+        href="https://forum.metagame.wtf/c/quest/5/l/latest?board=default"
+        isExternal
+      >
+        <Image src={MetaRaids} alt="MetaRaids" />
+        Raids
+      </MenuItem>
+
+      <Button
+        display="flex"
+        zIndex="11"
+        textDecoration="none"
+        variant="link"
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
```

Comment:
A lot of these props are being reused. MenuItem, SubMenuItem, and this all look like they are doing the same thing. Should consolidate the props and compose the components instead of repeating them, code will look much cleaner that way as well.

A good rule of thumb is to make things like MenuItem and SubMenuItem as generic and reusable as possible and declare them in their own files so that other people can also see it and re-use it. Defining one of specific components in files like this leads to a lot of repeated code which makes it hard to maintain and introduces inconsistencies in the UX/UI

Reply:
@pakokrew, helped resolve this and created a `utils` folder!